apps/to-do-api/host.json

```diff
@@ -2,9 +2,21 @@
   "version": "2.0",
   "logging": {
     "logLevel": {
-      "default": "Information"
+      "default": "Warning",
+      "Function": "Error",
+      "Host.Aggregator": "Error",
+      "Host.Results": "Information"
+    },
+    "applicationInsights": {
```

Comment:
consider that this section, which is specific for the host and does not affect worker, is ignored when `"telemetryMode": "OpenTelemetry"`